File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-java/incoder-6/shortest_paths.java
--- 
+++ 
@@ -12,7 +12,9 @@
                     weight_by_node.get(edge.get(0))
                             + weight_by_edge.get(edge),
                     weight_by_node.get(edge.get(1)));
-            weight_by_edge.put(edge, update_weight);
+            if (update_weight < weight_by_node.get(edge.get(1))) {
+                weight_by_node.put(edge.get(1), update_weight);
+            }
         }
     }
     return weight_by_node;

File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-python/incoder-1/shortest_paths.py
--- 
+++ 
@@ -6,7 +6,8 @@
 
     for i in range(len(weight_by_node) - 1):
         for (u, v), weight in weight_by_edge.items():
-            weight_by_edge[u, v] = min(
+            weight_by_node[v] = min(
+                weight_by_node[v],
                 weight_by_node[u] + weight,
                 weight_by_node[v]
             )
